### ISSUE TITLE
MEDIUM: Exchange revaluation total_gain/total_loss only computed for summary ledger

### ISSUE THREAD

**Issue by williamjmorenor** [cacao-accounting/cacao-accounting#665]
## Summary

In `ExchangeRevaluationService.run()`, the `total_gain` and `total_loss` fields on the `ExchangeRevaluation` record are computed by filtering entries where `ledger_id == summary_ledger.id`. However, the revaluation creates GL entries for **ALL** active ledgers (books), not just the summary ledger. The reported gain/loss totals only reflect the entity-currency (summary) book, silently omitting FX differences recorded in secondary books.

## Location

`cacao_accounting/contabilidad/exchange_revaluation_service.py` — `run()` lines 174-191

## Root Cause

```python
run.total_gain = sum(
    (
        self._decimal(entry.credit)
        for entry in entries
        if entry.ledger_id == summary_ledger.id  # Only summary ledger
        and entry.account_id == defaults.unrealized_exchange_gain_account_id
    ),
    Decimal("0"),
)
run.total_loss = sum(
    (
        self._decimal(entry.debit)
        for entry in entries
        if entry.ledger_id == summary_ledger.id  # Only summary ledger
        and entry.account_id == defaults.unrealized_exchange_loss_account_id
    ),
    Decimal("0"),
)
```

The `entries` list contains entries for ALL ledgers (created in `_build_entries_and_items` → `_entries_for_draft` which sets `ledger_id=draft.ledger.id`). But the gain/loss totals only sum entries for the summary ledger. If there are secondary books with different currencies that also have FX differences, those amounts are excluded from the reported totals.

## Reproduction

1. Configure a company with two books: primary (HNL) and secondary (USD)
2. Create open monetary items in both books with foreign currency exposure (e.g., EUR receivables)
3. Run exchange revaluation
4. Observe:
   - GL entries are created in BOTH books
   - `run.total_gain` and `run.total_loss` only reflect the summary ledger (HNL book)
   - The USD book's FX differences are not included in the totals
5. Report shows incorrect total gains/losses

## Impact

- Exchange revaluation gain/loss totals are incorrect for multi-book companies
- Dashboard and reporting show wrong FX impact
- Affects R2R (Record to Report) flow — exchange revaluation
- The GL entries themselves are correct; only the summary totals are wrong

## Recommendation

Remove the `entry.ledger_id == summary_ledger.id` filter from the `total_gain` and `total_loss` computations to include all ledgers.
